wbooks/src/app/index.tsx

```diff
@@ -1,38 +1,26 @@
 import 'react-native-gesture-handler';
-import React, { useEffect } from 'react';
-import { SafeAreaView, StatusBar, View, Text } from 'react-native';
-import { Provider, useDispatch } from 'react-redux';
-import actionCreators from '@redux/books/actions';
+import React from 'react';
+import { SafeAreaView, StatusBar } from 'react-native';
+import { Provider } from 'react-redux';
+import { colorsApp } from '@constants/colorsApp';
 
 import store from '../redux/store';
 
-function App() {
-  // eslint-disable-next-line no-console
-  console.tron.log('Hello world!!!');
+import styles from './styles';
+import AppNavigator from './components/AppNavigator';
 
+function App() {
   return (
     <>
-      <StatusBar barStyle="dark-content" />
-      <SafeAreaView>
+      <StatusBar backgroundColor={colorsApp.primary} barStyle="light-content" />
+      <SafeAreaView style={styles.safeAreaTop} />
+      <SafeAreaView style={styles.safeAreaBottom}>
```

Comment:
What are these SafeAreaViews for? You can remove them and add the styles you need to the specific screens